src/libostree/ostree-repo-pull.c

```diff
@@ -1322,6 +1324,165 @@ commitstate_is_partial (OtPullData   *pull_data,
 }
 
 static gboolean
+ensure_collection_ref_in_summary (GVariant   *summary,
+                                  const char *collection_id,
+                                  const char *ref)
+{
+  g_assert_nonnull (collection_id);
+  g_assert_nonnull (ref);
+
+  g_autoptr(GVariant) additional_metadata = NULL;
+  additional_metadata = g_variant_get_child_value (summary, 1);
```

Comment:
New ostree code style is to init variables with declaration.  (Applies to other places below)